docs/reference/cli-reference/cli-ref-pack.md

```diff
@@ -14,7 +14,7 @@ ms.topic: reference
 Creates a NuGet package based on the specified [.nuspec](../nuspec.md) or project file. The `dotnet pack` command (see [dotnet Commands](../dotnet-Commands.md)) and `msbuild -t:pack` (see [MSBuild targets](../msbuild-targets.md)) may be used as alternates.
 
 > [!Important]
-> Use [`dotnet pack`](../dotnet-Commands.md) or [`msbuild -t:pack`](../msbuild-targets.md) for [PackageReference](../../consume-packages/package-references-in-project-files.md) based projects.
```

Comment:
I think this change may be read as a little misleading. 
How about flipping the order:

> Use [`dotnet pack`](../dotnet-Commands.md) or [`msbuild -t:pack`](../msbuild-targets.md) for [PackageReference](../../consume-packages/package-references-in-project-files.md) based projects. Starting with NuGet version 6.5+, the pack command will error out when attempting to pack these project types.

Reply:
Please note, before with 6.5+ these project types were not fully supported, only silent about it.

Reply:
Great point! Maybe:

> Use [`dotnet pack`](../dotnet-Commands.md) or [`msbuild -t:pack`](../msbuild-targets.md) for [PackageReference](../../consume-packages/package-references-in-project-files.md) based projects. Starting with NuGet version 6.5+, the pack command will error out when attempting to pack these project types. Earlier versions would attempt to pack, but the packages were not correct.

Or something along those lines :)